daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md

```diff
@@ -73,6 +73,72 @@ It also includes a `RunAsync` method that does the heavy lifting of the workflow
     }
 ```
 
+{{% /codetab %}}
+
+{{% codetab %}}
+
+<!--python-->
+
+[In the following example](todo), for a basic Python order processing application using the Python SDK, your project code would include:
```

Comment:
The code examples between the .NET client and the Python client are not the same level. One is writing new workflows with activities, whilst the other one is at the level of the workflow management APIs. The code examples should be consistent between the two languages

Reply:
went ahead and removed for now since the python code is still in development